content/post/2022-10-07-friday-hacks-229.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Friday Hacks #229, October 7: How Supabase uses Cloudflare Workers and How to Build an Exchange"
+date: 2022-09-30 16:13:14.5144
+author: Shen Yi Hong
+url: /2022/10/friday-hacks-229
+nofh: false
+---
+
+**Date/Time:** Friday, October 7 at 7:00pm SGT<br />
+**Venue:** [**The Hangar** @ i3 Building](https://goo.gl/maps/aUMwWptKFsajR93b9)<br />
```

Comment:
would Level 2 Audi instead of the hangar be clearer

Reply:
good point!